docs/docs/guides/labs/freshness/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Freshness Policies'
+sidebar_position: 100
+unlisted: True
+---
+import Preview from '@site/docs/partials/\_Preview.md';
```

Comment:
Instead of using the generic Preview partial, I recommend creating a freshness policy-specific partial, similar to the [dg & Components Preview partial](https://github.com/dagster-io/dagster/blob/master/docs/docs/partials/_DgComponentsPreview.md?plain=1), so you can add a more specific call to action -- I assume in this case, we want folks to reach out to their CSM if they have questions or feedback.